proposals/NNNN-optimize-connection-wait-range.md

```diff
@@ -0,0 +1,46 @@
+# Feature name
+
+* Proposal: [SDL-NNNN](NNNN-filename.md)
+* Author: [Timur Pulathaneli](https://github.com/tpulatha)
+* Status: **Awaiting review**
+* Impacted Platforms: [iOS]
+
+## Introduction
```

Comment:
The introduction is meant be 1-2 sentences describing the actual change, which will then go into the PR's first text field. You have put motivation text into the introduction.